VideoLocker/build.gradle

```diff
@@ -84,20 +84,28 @@ dependencies {
     }
     compile fileTree(dir: 'libs', include: '*.jar')
     compile 'com.facebook.android:facebook-android-sdk:3.22.+'
-    compile 'com.google.android.gms:play-services:6.5.+'
+    compile 'com.google.android.gms:play-services:7.5.0'
```

Comment:
Not your fault, but while we're at it, would you mind changing this to only include the APIs we actually need, instead of ALL of them?

There's a list of individual Play APIs at https://developers.google.com/android/guides/setup